lib/socket.dart

```diff
@@ -46,7 +46,7 @@ class PhoenixSocket<T> {
   }) async =>
       PhoenixSocket<T>._(
         encoding: encoding,
-        ws: await WebSocket.connect('$address/websocket'),
```

Comment:
This seems like an unrelated—and seemingly undesired—change.

Reply:
Unrelated, yes. Undesired, no. It prevents the use of query parameters on the websocket address. That said, I'll bring this up in another PR.

Reply:
Ohhhhh, right. We discussed this earlier. Forgot.

We should probably still take care of the `/websocket` part, if only because it's a non-obvious assumption of Phoenix. 🤷‍♂️

Reply:
The config endpoint now provides this full url, and it includes the `/websocket` part.